atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -374,7 +396,7 @@ def add_log_type(output):
         metadata_version = 0
 
         worker_metadatas = self.workflow.postbuild_results.get(PLUGIN_FETCH_WORKER_METADATA_KEY)
-        maven_components = get_maven_components(self.workflow)
+        remote_source_file_outputs, kojfile_components = get_maven_metadata(self.workflow)
```

Comment:
can we name 'kojifile_components' instead of 'kojfile_components' ?

Reply:
corrected typo